shared/desktop/CHANGELOG.txt

```diff
@@ -1,6 +1,2 @@
-Chat:
-• Replies will automatically notify who you're replying to
-
-Misc:
-• Configure a HTTP or SOCKS proxy in advanced settings
-• Support user lookup by phone number
+• Keybase now supports custom emoji
```

Comment:
How about adding some stock emoji here?

```suggestion
• Keybase now supports custom emoji ✨🔥🤯
```

Reply:
this is disallowed in the app store and i don't want to test that it works on windows etc